api/views/user_views.py

```diff
@@ -37,6 +39,20 @@
 import json
 
 
+def mailInstructor(email):
+    message = Mail(from_email="edukona.team@gmail.com", to_emails=email)
+
+    message.template_id = os.getenv("WELCOME_TEMPLATE_ID")
+    try:
+        sg = SendGridAPIClient(os.environ.get("SENDGRID_API_KEY"))
+        response = sg.send(message)
+        print(response.status_code)
+        print(response.body)
+        print(response.headers)
```

Comment:
This will have to be replaced by logging, but we can do that once the logging is set up.